{"uuid":"9c2c7c87-036a-4059-8aea-fdb0d1b68b9e","historyId":"9a81de9b3b5db9b8e8c7ab0c012e0fd8","fullName":"io.cucumber.testng.AbstractTestNGCucumberTests.runScenario","labels":[{"name":"package","value":"testRunner.DsalgoRunner"},{"name":"testClass","value":"testRunner.DsalgoRunner"},{"name":"testMethod","value":"runScenario"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"testRunner.DsalgoRunner"},{"name":"host","value":"LAPTOP-2KQV9F30"},{"name":"thread","value":"14292@LAPTOP-2KQV9F30.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Runs Cucumber Scenarios","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Console output is empty","trace":"java.lang.AssertionError: Console output is empty\r\n\tat org.junit.Assert.fail(Assert.java:89)\r\n\tat org.junit.Assert.assertTrue(Assert.java:42)\r\n\tat org.junit.Assert.assertFalse(Assert.java:65)\r\n\tat stepDefinitions.ArraysStep.the_user_gets_error_message_error_occurred_during_submission_in_the_console(ArraysStep.java:378)\r\n\tat âœ½.The user gets error message Error occurred during submission in the console(file:///C:/Workspace/Ds-Algo2025/src/test/resources/Feature/Arrays.feature:81)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg1","value":"\"Arrays PracticeQuestion\""},{"name":"arg0","value":"\"Verify that user receives error on submitting invalid python code for \"Search the array\" question\""}],"start":1746232466772,"stop":1746232472329}